tasks/post-bigquery-metrics-task.yaml

```diff
@@ -105,7 +105,8 @@ spec:
           sleep "${delay}"
         fi
 
-        if bq query \
+        # todo: un-negate
```

Comment:
LGTM, pending un-negating of this condition.

Reply:
Done. It's un-negated in [`2434cbe` (#44)](https://github.com/stackrox/konflux-tasks/pull/44/commits/2434cbe6de9451b15b79e3822f7a3ca71c582e6f).